content/webapp/services/prismic/transformers/events.ts

```diff
@@ -121,7 +121,7 @@ export function transformEvent(
 
   const matchedId =
     data.eventbriteEvent && data.eventbriteEvent.embed_url
-      ? /\/e\/([0-9]+)/.exec(data.eventbriteEvent.embed_url)
+      ? /\/e\/(.+)/.exec(data.eventbriteEvent.embed_url)
```

Comment:
1. Can we include some examples of the sort of Eventbrite ID we're trying to match as comments here?
2. Are we trying to capture everything after the `/e/`, or just the numeric component?

Reply:
Actually, I wonder if it would be worth pulling this out into a utility function `getEventbriteId(url: string | undefined): string | undefined` and throwing some unit tests around it?

Reply:
We're trying to match everything after the `/e/`. Historically, I think this was only a number, which is I imagine why we only included numbers.

I'll write some tests.